css/screens/products.css

```diff
@@ -0,0 +1,53 @@
+/* css for produts */
```

Comment:
Liked the way you are using a clean structure and well-commented sections. Keep up the good work.